object_tracker/object_tracker.py

```diff
@@ -44,11 +45,12 @@ def run_frame(self, img):
             self._match.roi = min_x, min_y, w, h
         flag = 1 if flag == "normal" else 0
         # x, y, w, h, was_frame_situation_algo_wise_was_normal, score
-        data = [min_x, min_y, max_x-min_x, max_y-min_y, flag, score]
+        data = [min_x, min_y, max_x - min_x, max_y - min_y, flag, score]
         return img, data
 
     def init_bounding_box(self, frame, bounding_box):
         self._tracker.init_tracker(frame, bounding_box)
+        logging.info("finish init tracker bounding box")
```

Comment:
Maybe add the bounding box shape/coordinates as well.

Reply:
done